client/dashboard/components/site-preview-link/index.tsx

```diff
@@ -87,31 +78,31 @@ const SitePreviewLink = ( {
 	disabled,
 	siteUrl,
 }: SitePreviewLinkProps ) => {
-	const translate = useTranslate();
-
-	let linkValue = `${ trailingslashit( siteUrl ) }?share=${ code }`;
+	let linkValue = `${ siteUrl }?share=${ code }`;
 	if ( isCreating ) {
-		linkValue = translate( 'Loading…' );
+		linkValue = __( 'Loading…' );
 	} else if ( isRemoving ) {
-		linkValue = translate( 'Disabling…' );
+		linkValue = __( 'Disabling…' );
 	}
 
 	const hasExpiration = expires_at && expires_at.length > 0;
 	return (
-		<>
-			<ClipboardButtonInput key={ code } value={ linkValue } disabled={ disabled } />
-			<HelpText>
-				{ hasExpiration ? (
+		<ClipboardInputControl
+			key={ code }
+			value={ linkValue }
```

Comment:
We need to add a visually hidden label to ensure the tooltip displays correctly. Additionally, it would be better to show a confirmation notice when the content is copied.

Reply:
Oh you're right. Done!